lib/winapi/debug.c

```diff
@@ -1,8 +1,14 @@
-#include <winbase.h>
+#include <windows.h>
 #include <winnt.h>
 
 #include <string.h>
 
+BOOL IsDebuggerPresent (VOID)
+{
+    // FIXME: Can we detect debugger presence on the Xbox?
+    return FALSE;
```

Comment:
"winapi: Implement IsDebuggerPresent() "

Commit title is bad; this merely stubs it.

Reply:
Fixed.